src/main/java/rx/internal/operators/OperatorFinally.java

```diff
@@ -33,6 +33,9 @@
     final Action0 action;
 
     public OperatorFinally(Action0 action) {
+        if (action == null) {
```

Comment:
This operator will be used in `Single.finallyDo()`, I don't want to copy-paste this :)